asserts/device_asserts.go

```diff
@@ -81,6 +81,11 @@ func (mod *Model) Kernel() string {
 	return mod.HeaderString("kernel")
 }
 
+// KernelTrack returns the kernel track the model uses.
```

Comment:
I think we want also a call to checkOptionalString(assert.headers, "kernel-track") in assembleModel

Reply:
@pedronis fixed and pushed, please have a look